### PR TITLE
Jv 6 music on screens

### LINKED ISSUE #6: Music on different screens
**Is your feature request related to a problem? Please describe.**
The App is too quiet so depending on witch screen you are on, different music will play.

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
_List Changes Introduced by this PR_
1. Added theme songs to Pokedex screens.
2. Styled sound button.

## Purpose
To play different Pokemon music depending on use case.

## Approach
Used audio tag with Ref in order to change volume and play music.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #6

### CODE REVIEW

File: pokemon-app/src/components/EnemyPokemon.js [hunk 1]
```diff
@@ -1,41 +1,94 @@
...
+              }`}
+            >
+              <ol className="enemy-pokemon__moves-list">
+                {!enemyPokemon?.moves
+                  ? null
+                  : enemyPokemon?.moves.map((moveData, moveIndex) => (
+                      <li key={moveIndex}>{moveData.move.name}</li>
+                    ))}
+              </ol>
+            </div>
```
Comment (Lap343) [inline pokemon-app/src/components/EnemyPokemon.js]: weird it has my code

File: pokemon-app/src/components/EnemyPokemon.js [hunk 2]
```diff
@@ -1,41 +1,94 @@
...
+            <div className="enemy-pokemon-types">
+              <Types pokemon={enemyPokemon} />
+            </div>
+
+            <div
+              onClick={() => setEnemyStatsOnTop(true)}
+              className={`enemy-pokemon__stats-title ${
+                enemyStatsOnTop ? "on-tip-top" : ""
+              }`}
+            >
```
Comment (kmausolf) [inline pokemon-app/src/components/EnemyPokemon.js]: Stuff like this makes a great use case for styled components.  With styled components, you can adjust the styling based on props.

File: pokemon-app/src/components/Pokeball.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from "react";
+import "../styles/Pokeball.css";
+
+const Pokeball = ({ isPokeball, setIsPokeball }) => {
+  useEffect(() => {
+    if (isPokeball) {
+      const timer = setTimeout(() => setIsPokeball(false), 1700);
+      return () => clearTimeout(timer);
+    }
```
Comment (kmausolf) [inline pokemon-app/src/components/Pokeball.js]: Nice use of cleanup functions!

File: pokemon-app/src/components/ThemeSongs.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    }
+  }
+}
+
+useEffect(() => {
+  if (audioPlayer.current) {
+    audioPlayer.current.volume = 0.1;
+    audioPlayer.current.play();
+  }
+}, []);
```
Comment (kmausolf) [inline pokemon-app/src/components/ThemeSongs.js:26]: Great implementation of this.  Couldn't ask for more.

**Review by Lap343 — APPROVED** [review]
Very nice you guys! Really good job on this head scratcher.

File: pokemon-app/src/components/EnemyPokemon.js (hunk 1 above)
Comment (gonzalespaulb) [reply pokemon-app/src/components/EnemyPokemon.js]: Excited to see the final product

**Review by krferrell — APPROVED** [review]
Well done! It looks very clean and comfortable for the user's ears

**Review by JayDampitan — APPROVED** [review]
I see yall got some sound. Very nice. cant wait for it.

**Pull request merged**